app/Http/Controllers/Wiki/LanguageControllerV2.php

```diff
@@ -265,11 +265,12 @@ function () use ($root, $iso, $media, $full_word, $organization_id) {
                     ->includeAutonymTranslation()
                     ->filterableByIsoCode($iso)
                     ->filterableByName($root, $full_word)
-                    ->when($organization_id, function ($query) use ($organization_id) {
-                        return $query->whereHas('filesets', function ($q) use ($organization_id) {
-                            $q->where('organization_id', $organization_id);
-                        });
-                    })->when($media, function ($query) use ($media) {
+                    // ->when($organization_id, function ($query) use ($organization_id) {
+                    //     return $query->whereHas('filesets', function ($q) use ($organization_id) {
+                    //         $q->where('organization_id', $organization_id);
+                    //     });
+                    // })
```

Comment:
Should this comment be removed?

Reply:
Hi @mgonzalezs I may need this code later. The first pass through is to verify the path and model are supported. Then later I will evaluate if all the high-usage request params are supported. I may need to resurrect this code and fix it. Do you mind if we keep it in for now?

Reply:
I do not, but maybe adding a TODO comment just in case another dev comes back here.